src/Clustering/GMUClusterManager.h

```diff
@@ -45,12 +50,19 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface GMUClusterManager : NSObject<GMSMapViewDelegate>
 
-/** Initializer with the |mapView|, |algorithm| and |renderer|. */
-- (instancetype)initWithMap:(GMSMapView *)mapView
-                  algorithm:(id<GMUClusterAlgorithm>)algorithm
-                   renderer:(id<GMUClusterRenderer>)renderer;
+/**
+ * The default initializer is not available. Use initWithMap:algorithm:renderer instead.
+ */
+- (instancetype)init NS_UNAVAILABLE;
+
+/**
+ * Returns a new instance of the GMUClusterManager class defined by it's |algorithm| and |renderer|.
+ */
+- (instancetype)initWithMap:(GMSMapView *)mapView algorithm:(id<GMUClusterAlgorithm>)algorithm renderer:(id<GMUClusterRenderer>)renderer NS_DESIGNATED_INITIALIZER;
```

Comment:
Please wrap this as in the original declaration. This will allow lines to fit within the 100 character limit and assist readability.